src/App.tsx

```diff
@@ -27,13 +28,30 @@ type ReorderOptions = {
 };
 
 // Use this function in the render method to prepare goods
-export function getReorderedGoods(
+function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  visibleGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case (1):
```

Comment:
I don't understand what to do

Reply:
use enum type, `case SortType.LENGTH:`, for example instead of `case (2):` and this.setState({ sortType: SortType.ALPHABET }); instead of this.setState({ sortType: 1 });